zkwrapper/zk_queue.cc

```diff
@@ -0,0 +1,62 @@
+//
+// Created by Prudhvi Boyapalli on 10/17/16.
+//
+
+#include "zk_queue.h"
+#include <easylogging++.h>
+
+const std::string ZKQueue::CLASS_NAME = ": **ZkQueuer** : ";
+
+std::string q_path;
+std::string peek_path;
+std::string element;
+
+std::string ZKQueue::peek() {
+    return peek_path;
+}
+
+int ZKQueue::pop() {
+    int error_code;
+    if (zk.delete_node(peek_path, error_code)) {
+        LOG(ERROR) << CLASS_NAME <<  "The element '"<< peek_path << "' was popped.";
+        std::vector<std::string> children = std::vector<std::string>();
+        // TODO: Don't set children
```

Comment:
Not sure, looks like it showed up in this commit by @Vegetable26 

https://github.com/Rice-Comp413-2016/RDFS/commit/736b4b3f6393319f9c63beffd24b2feb6f87421c